src/components/renderer/RenderableAttachmentRenderer.tsx

```diff
@@ -1,14 +1,15 @@
 import { PdfRenderer } from "./PdfRenderer";
 import React from "react";
 import { Attachment } from "../../types";
+import { isV2Attachment } from "../../utils";
 
 /**
  * Function returning the correct attachment renderer depending on the attachment type. Currently supports:
  * - application/pdf
  * returns null when not attachment cannot be rendered
  */
 export function renderableAttachmentRenderer(attachment: Attachment): React.FunctionComponent | null {
-  if (attachment.type === "application/pdf") {
+  if (isV2Attachment(attachment) ? attachment.type === "application/pdf" : attachment.mimeType === "application/pdf") {
```

Comment:
just so that you feel I reviewed, I will leave a comment. Maybe can create a getMimeType function like getTemplateName

Reply:
just so that you feel I made changes, I've added `getAttachmentMimeType`.